README.md

```diff
@@ -62,3 +62,106 @@ happy_r_google_api:
 
 
 [1]: https://github.com/google/google-api-php-client
+
+Basic Usage
+------------
+
+### Step 1: Create a controller
+Create a controller with `authenticate` and `redirect` methods.
+
+```
```

Comment:
Add "php" add the end of the line. You will get better syntax highlighting then.